### PR TITLE
Update auth view controller

### PR DESCRIPTION
**Pull request by boxtcarpel**
Expose some advanced configuration options

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @boxtcarpel has signed the CLA. Thanks for the pull request!

File: BoxContentSDK/BoxContentSDK/OAuth2/BOXAuthorizationViewController.h [hunk 1]
```diff
@@ -30,14 +31,36 @@
  */
 @interface BOXAuthorizationViewController : UIViewController
 
+/**
+ * The authroziation URL to be used.
```
Comment (sowmiya) [inline BoxContentSDK/BoxContentSDK/OAuth2/BOXAuthorizationViewController.h:35]: spelling.

**Comment by sowmiya** [comment]
I dont fully understand why we need to expose a setter for authorizationURL, would love to catch up on that. Other than that it is a +2.

**Comment by boxtcarpel** [comment]
It's mainly for the ability to have fall-back auth URLs, we may remove it.

**Pull request merged**